syntax = "proto3";

service UserServices{
    rpc RegisterUser (registerUserRequest) returns (registerUserResponse) {};
}

message registerUserRequest{
    string username = 1;
    string email = 2;
    string password = 3;
    bool admin = 4;
}

message registerUserResponse{
    string message = 1;
    string email = 2;
}